### ISSUE TITLE
Network and Graph2d not working in the same app

### ISSUE THREAD

**Issue by richardeaxon** [visjs/vis-timeline#67]
We use both network and 2d charts in the same app. This worked fine in vis 4.x but no longer works. I get the following error:

`TypeError: vis.Network is not a constructor`

If the load order is 

```
        <!--Vis-->
        <script src="dist/vis-network.min.js"></script>
        <script src="dist/vis-timeline-graph2d.min.js"></script>
```
and 

`TypeError: vis.Graph2d is not a constructor`

if the graph2d module loads first. It appears like the one module overwrites the other. The last module in the load order is the one that works.

**Comment by mojoaxel** [comment]
> Vis Timeline issue would be more appropriate as there doesn't seem to be anything wrong with Vis Network.

I moved this issue to vis-timeline.

**Comment by Thomaash** [comment]
@mojoaxel We need peer dependencies for this to work properly, see https://github.com/visjs/vis-network/pull/85 . There is an unreleased partial fix (#53) but that doesn't resolve all of the issues.

**Comment by yotamberk** [comment]
@Thomaash What about this issue? What else needs to be done for this?

**Comment by Thomaash** [comment]
I need to be reminded about this and do the same I did in Vis Network everywhere else.

**Comment by richardeaxon** [comment]
Would be really nice to have a working peer build for graph2d. 

https://unpkg.com/vis-timeline@latest/peer/umd/vis-timeline-graph2d.min.js
Cannot find "/peer/umd/vis-timeline-graph2d.min.js" in vis-timeline@6.5.2

So I am still stuck with the old visjs as I cannot use vis-network and vis-graph2d together.

**Comment by Thomaash** [comment]
Hi @richardeaxon,

the fix is already in #269. It just needs to be reviewed to make sure everything's fine and can be released.

**Comment by richardeaxon** [comment]
> the fix is already in #269. It just needs to be reviewed to make sure everything's fine and can be released.

That would be awesome. Thanks.
